src/main/webapp/WEB-INF/static/js/trialmanager/manageTrial.js

```diff
@@ -827,8 +827,11 @@ showAlertMessage,showMeasurementsPreview,createErrorNotification,errorMsgHeader,
 
 						scope.forkPrintLabelFlows = function () {
 							if (studyContext.measurementDatasetId === scope.selected.datasetId) {
-								// Old workflow for plot dataset. TODO migrate
-								createLabelPrinting();
```

Comment:
delete old method/files? I'm not sure if there's still a workflow that uses the old method, I think this was the last one

Reply:
we still have field map, let's see what they decide today